src/app/code/community/Sitewards/Webappmanifest/Model/Manifest.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * @category    Sitewards
+ * @package     Sitewards_Webappmanifest
+ * @copyright   Copyright (c) Sitewards GmbH (http://www.sitewards.com/)
+ */
+class Sitewards_Webappmanifest_Model_Manifest extends Mage_Core_Model_Abstract

```

Comment:
I'm not sure a model should be responsible for returning the serialised format. In my mind, the responsibilities are thusly:

model: fetch the data
controller: request serialisation

The model might be able to implement some sort of "express me as json" interface which would allow Zend_Json::ecode($oModel)?